src/server/service/mail/remindReviewers.ts

```diff
@@ -1,72 +1,141 @@
+    const values = {
+      assessmentName: t(`assessment.${assessmentName}`),
+      cycleName,
+      countries: countries.map((country) => _getCountryName(country, t)).join(', '),
+      countryUrls: countries.map((country) => _getCountryUrl(country.countryIso, assessmentName, cycleName)).join('\n'),
+      countryLinks: countries.map((country) => _getCountryLink(assessment, cycle, country, t)).join('<br>'),
+    }
 
-  const subject = t('email.remindReviewer.subject', values)
-  const text = t('email.remindReviewer.textMessage', values)
-  const htmlStyle = `style="white-space: pre-line; max-width: 100%"`
-  const html = `<p ${htmlStyle} >${t('email.remindReviewer.htmlMessage', values)}</p>`
-  return { to, subject, text, html }
-}
+    messageBodyHTML += t('email.remindReviewer.messageBodyHTML', values)
+    messageBodyText += t('email.remindReviewer.messageBodyText', values)
+  })
+
+  const text = `${messageHeader}\n\n${messageBodyText}\n\n${messageFooter}`
+  const html = `<p ${htmlStyle} >${messageHeader}${messageBodyHTML}${messageFooter}</p>`
 
-const getRecipients = async (props: { cycle: Cycle; countryIso: CountryIso }) => {
-  const { countryIso, cycle } = props
-  const roles = UserRoles.getRecipientRoles({ status: AssessmentStatus.review })
+  const mail = { to, subject, text, html }
+
+  return mail
+}
 
-  return UserRepository.readCountryUsersByRole({ countryISOs: [countryIso], cycle, roles })
+const getReviewerRecipients = async (props: {
+  assessments: Array<Assessment>
+}): Promise<RecordRecipientAssessmentCycleCountries> => {
+  const { assessments } = props
+  const assessmentsByReviewer: RecordRecipientAssessmentCycleCountries = {}
+
+  await Promise.all(
+    assessments.map(async (assessment) => {
+      return Promise.all(
+        assessment.cycles.map(async (cycle) => {
+          const countries = await AreaController.getCountries({ assessment, cycle })
+          const inReview = countries.filter((country) => {
+            const diffInDays = Dates.differenceInDays(new Date(), new Date(country.lastInReview))
+            return (
+              country.props.status === AssessmentStatus.review &&
+              diffInDays > 6 &&
+              diffInDays % 7 === 0 &&
+              !Areas.isAtlantis(country.countryIso)
+            )
+          })
+
+          if (!inReview.length) return
+
+          const users = await UserRepository.readCountryUsersByRole({
+            countryISOs: inReview.map((c) => c.countryIso),
+            cycle,
+            roles: UserRoles.getRecipientRoles({ status: AssessmentStatus.review }),
+          })
+
+          users.forEach((user) => {
+            if (!assessmentsByReviewer[user.email]) {
+              assessmentsByReviewer[user.email] = {
+                user,
+                assessments: [],
+              }
+            }
+
+            assessmentsByReviewer[user.email].assessments.push({ assessment, cycle, countries: inReview })
+          })
```

Comment:
We construct an object of 

`[user.email]: { user, assessments: [ { assessment, cycle, countries } ] }`